components/common/Layout/Layout.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from "react";
+import styles from "./layout.module.css";
+import Navbar from "../Navbar/Navbar";
+import Footer from "../Footer/Footer";
+
+export default function Layout({ children }: { children: ReactNode }) {
```

Comment:
이렇게 적용해주거나 일관되게 별도 타입을 정의해주면 좋을 것 같아요!
```suggestion
export default function Layout({ children }: PropsWithChildren) {
```